src/create_disk.sh

```diff
@@ -29,6 +29,8 @@ Options:
     --disk: disk device to use
     --help: show this help
     --kargs: kernel CLI args
+    --platform: Ignition platform ID
```

Comment:
Minor: `--platform-id` would be more canonical

Reply:
Hmm, I'm not seeing any instances of `--platform-id` in the codebase.  `kola run` already has `--platform`, as does `cosa artifact-disk manual`.

Reply:
Hmm right. I didn't mean literally `--platform-id` the switch, but platform ID the name. But indeed, we're a bit inconsistent about this between switches and documentation/internal variable names, etc...